src/Components/test/E2ETest/Tests/SectionsTest.cs

```diff
@@ -0,0 +1,78 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using BasicTestApp;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.E2ETesting;
+using Xunit.Abstractions;
+using Microsoft.AspNetCore.Components.E2ETest;
+using OpenQA.Selenium;
+
+namespace Microsoft.AspNetCore.Components.E2ETests.Tests;
+public class SectionsTest : ServerTestBase<ToggleExecutionModeServerFixture<Program>>
+{
+    private IWebElement _appElement;
+
+    public SectionsTest
+        (BrowserFixture browserFixture,
+        ToggleExecutionModeServerFixture<Program> serverFixture,
+        ITestOutputHelper output)
+        : base(browserFixture, serverFixture, output)
+    {
+    }
+
+    protected override void InitializeAsyncCore()
+    {
+        Navigate(ServerPathBase, noReload: _serverFixture.ExecutionMode == ExecutionMode.Client);
```

Comment:
I just found out while working on E2E tests for QuickGrid that `noReload` was removed in https://github.com/dotnet/aspnetcore/pull/35414 and setting it doesn't have any effect. @SteveSandersonMS was there a reason to keep it as an argument for `WebDriverExtensions.Navigate()`?

Reply:
@Nick-Stanton thanks! I'll investigate this.

Reply:
I opened an issue for this https://github.com/dotnet/aspnetcore/issues/46810